infra/terraform/terraform.go

```diff
@@ -270,6 +270,7 @@ func (r *terraform) boot(ctx context.Context) (output string, err error) {
 		"-var", fmt.Sprintf("nodes=%d", r.NumNodes),
 		"-var", fmt.Sprintf("os=%s", r.OS),
 		"-var", fmt.Sprintf("random_password=%s", uuid.NewV4().String()),
+		"-var", fmt.Sprintf("installer_url=%s", r.InstallerURL),
```

Comment:
Pass installer_url variable to terraform, because it can be used in client's bootstrap script.

Reply:
Why should one transfer installer using bootstrap script? 
robotest has sufficient support for all key installer transfer methods (s3, http and localfile)